pkg/codegen/testing/test/testdata/dashed-import-schema/python/pulumi_plant/_inputs.py

```diff
@@ -31,12 +31,15 @@ def __init__(__self__, *,
     @staticmethod
     def _configure(
              _setter: Callable[[Any, Any], None],
-             size: pulumi.Input['ContainerSize'],
              brightness: Optional[pulumi.Input['ContainerBrightness']] = None,
              color: Optional[pulumi.Input[Union['ContainerColor', str]]] = None,
              material: Optional[pulumi.Input[str]] = None,
              opts: Optional[pulumi.ResourceOptions]=None,
              **kwargs):
+        if 'size' in kwargs:
+            size = kwargs['size']
+        if 'size' not in locals():
+            raise TypeError("Missing required property 'size'")
```

Comment:
When the required argument is removed the `if 'size' not in locals()` check is added as a fallback.